recipes/dsm2dtm/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "dsm2dtm" %}
+{% set version = "0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/seedlit/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 64a08fd43eea5c98cd7b7d4426f7646ad989e607ceb104047348b75d2aa2584b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  build:
+    - git
+    - python
+    - pip
+    - setuptools
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - gdal >=3.0.*
+    - rasterio >=1.2.*
+    - numpy >=1.20.*
+    - certifi
+
+test:
+  imports:
+    - dsm2dtm
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/seedlit/dsm2dtm
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Generates DTM (Digital Terrain Model) from DSM (Digital Surface Model).
+  description: |
+    Generates DTM (Digital Terrain Model) from DSM (Digital Surface Model).
+  doc_url: https://github.com/seedlit/dsm2dtm#readme
+  dev_url: https://github.com/seedlit/dsm2dtm
+extra:
+  recipe-maintainers:
+    - seedlit
```

Comment:
Why are you not also listing yourself here @R1j1t ?

Reply:
Hey @dopplershift, I just thought if I will be contributing in the future,I will open a PR! :)

Reply:
Anyone contributing in the future should use a PR. With few exceptions, we don't push directly to the main/master branch.

Since you're contributing this recipe, it's standard that you should be listed as a maintainer.

Reply:
I understand, I will add my GitHub username.